app/views/campaigns/index.html.haml

```diff
@@ -1,21 +1,23 @@
+= hot_javascript_tag 'campaigns'
+
 .container.dashboard
   %header
     %h1
-      Campaigns
+      = t('campaign.campaigns')
+
 %section
   .container
     %hr
-    %h1
-      Create New Campaign
-    %div
-      =form_tag do
-        = text_field :campaign, 'title', placeholder: 'Title'
-        = submit_tag 'Create Campaign', class: 'button dark'
-%section
+    - if current_user && Features.open_course_creation?
```

Comment:
Admins on Wiki Ed dashboard also need to be able to create a campaign, even with open_course_creation disabled.